docs/platforms/react-native/sourcemaps/index.mdx

```diff
@@ -50,3 +50,7 @@ sentry {
     uploadNativeSymbols = shouldSentryAutoUpload()
 }
 ```
+
+### Debug and Simulator Builds
+
+Note that uploading source maps for debug or simulator builds is disabled because symbolication is handled directly by the Metro development server. In a development environment, Metro automatically provides the symbolication needed to translate stack traces into readable source code. As a result, there is no need to upload source maps for these types of builds.
```

Comment:
Release with simulator target will auto upload the source maps.

```suggestion
### Debug Builds

Note that automatic uploading of source maps for debug builds is disabled because symbolication is handled directly by the Metro development server. In a development environment, Metro automatically provides the symbolication needed to translate stack traces into readable source code. As a result, there is no need to upload source maps for these types of builds.
```